client/Windows/wf_cliprdr.c

```diff
@@ -1482,15 +1486,12 @@ static LRESULT CALLBACK cliprdr_proc(HWND hWnd, UINT Msg, WPARAM wParam, LPARAM
 				case OLE_SETCLIPBOARD:
 					DEBUG_CLIPRDR("info: OLE_SETCLIPBOARD");
 
-					if (S_FALSE == OleIsCurrentClipboard(clipboard->data_obj))
```

Comment:
Why is this line removed?

Reply:
when copy file A.txt from remote, then copy file B.txt, if add this line, will not place file B.txt in clipboard, because file A.txt clipboard data object is still in clipboard, so  OleIsCurrentClipboard(clipboard->data_obj) return S_OK.